### PR TITLE
Add new mob and coins + bug fix

### LINKED ISSUE #12: Zombies can hit from large distances on spawn
When a mob spawns (could also be when you get close to a dead zombie?) you take damage

https://user-images.githubusercontent.com/34372536/163901723-b23bfdd1-dea0-40cd-99ad-515c3cef690d.mp4

It may be of note that the knockback I receive comes from behind

### PR DESCRIPTION
**Pull request by kiip1**
- Zombie attack range fixed (Closes #12)
- Add coins (doesn't do anything yet)
- Add spider (throws webs) and baby variant of zombie
- Spread mobs around instead of spawning them all at one spot
- Bug fix (Death message)

### CODE REVIEW

File: src/main/java/net/minestom/arena/Main.java [hunk 1]
```diff
@@ -62,6 +63,9 @@ public static void main(String[] args) {
                 player.setEnableRespawnScreen(false);
             });
 
+            // Prevent dropping
+            handler.addListener(ItemDropEvent.class, event -> event.setCancelled(true));
```
Comment (TheMode) [inline src/main/java/net/minestom/arena/Main.java:67]: This listener should be specific to `LobbyInstance`

File: src/main/java/net/minestom/arena/game/Arena.java [hunk 2]
```diff
@@ -9,4 +9,6 @@ public interface Arena {
     @NotNull Group group();
 
     @NotNull CompletableFuture<Void> init();
+
+    int stage();
```
Comment (TheMode) [inline src/main/java/net/minestom/arena/game/Arena.java:13]: Is it necessary? Not all arena may be wave/stage-based

File: src/main/java/net/minestom/arena/group/Group.java [hunk 3]
```diff
@@ -10,6 +12,9 @@ static Group findGroup(@NotNull Player player) {
         return GroupManager.getGroup(player);
     }
 
+    void play(Arena arena);
```
Comment (TheMode) [inline src/main/java/net/minestom/arena/group/Group.java:15]: Very misleading as this does not do anything. Should be internal.

**Review by TheMode — CHANGES_REQUESTED** [review]

File: src/main/java/net/minestom/arena/Main.java (hunk 1 above)
Comment (kiip1) [reply src/main/java/net/minestom/arena/Main.java]: Fixed in latest commit

File: src/main/java/net/minestom/arena/game/Arena.java (hunk 2 above)
Comment (kiip1) [reply src/main/java/net/minestom/arena/game/Arena.java]: Fixed in latest commit

File: src/main/java/net/minestom/arena/group/Group.java (hunk 3 above)
Comment (kiip1) [reply src/main/java/net/minestom/arena/group/Group.java]: Fixed in latest commit

**Pull request merged**